cloudbuild_cosign.sh

```diff
@@ -0,0 +1,37 @@
+#!/bin/sh
+
+set -o errexit
+set -o xtrace
+
+
+export KMS_VAL=gcpkms://projects/$PROJECT_ID/locations/global/keyRings/cosign/cryptoKeys/cosign
+# TODO (priyawadhwa@): Store signatures in distroless repo once we are confident this works
+export COSIGN_REPOSITORY=gcr.io/distroless-signatures
+
+
```

Comment:
For debugging, could you throw a "cosign version" here too?

Reply:
fosh